### PR TITLE
Add import tests for the py3 standard library

### PR DESCRIPTION
**Pull request by hwright**
This also adds additional shared libraries to the python 3 image which
are required by some of the standard library.

The one standard library bit we don't test is the `tkinter` integration,
which brings in a bunch of dependencies, but doesn't seem very
relevant for deploying server-side containers.

### CODE REVIEW

**Comment by hwright** [comment]
Ping.

**Comment by hwright** [comment]
This also addresses #120 for the python 3 case.

File: python3/testdata/python3.yaml [hunk 1]
```diff
@@ -12,3 +12,583 @@ commandTests:
   - name: symlink 
     command: ["/usr/bin/python", "--version"]
     expectedOutput: ["Python 3.5.3"]
+
+  - name: import_abc
```
Comment (r2d4) [inline python3/testdata/python3.yaml]: @nkubala is there a simpler way to do this?

**Comment by hwright** [comment]
What are the next steps here?

**Comment by r2d4** [comment]
Sorry, LGTM.  My only question was whether we could take this and have a macro directly in the structure tests since it seems like a pattern that would be useful for other projects.

**Comment by hwright** [comment]
@r2d4 Yeah, the pattern is probably abstractable at some level, I just don't want that to be part of this yak shave.

Thanks for the review.

**Pull request merged**